docs/build-process.md

```diff
@@ -0,0 +1,75 @@
+# apko Build Process
+
+apko builds an OCI-compliant image, stored in a tar file, that can then be loaded into a container runtime.
+The entire build is configured from a declarative file `apko.yaml`, which does not allow the execution of arbitrary
+commands. Instead, all content is generated from installing `apk` packages.
+
+The build process is driven by the implementation of the `apko build` command, specifically
+[`BuildCmd()`](../internal/cli/build.go#L104).
+
+The entire build process involves laying out the desired files in a temporary working directory, and then
+packaging that working directory as an OCI filesystem layer `.tar.gz`. With that layer file in hand,
+it can be packaged into an OCI image, and an SBoM can be created.
+
+The process is as follows:
+
+1. Create a temporary working directory.
+1. Create a [`build.Context`](../pkg/build/build.go#L37-45). This `Context` contains:
+   * The path to the config file, default to `apko.yaml`
+   * The parsed config file into an internal structure [`ImageConfiguration`](../pkg/build/types/types.go#L55-83)
+   * The [`buildImplementation`](../pkg/build/build_implementation.go#L43-59), which is the engine responsible for executing the actual build
+   * The [`Executor`](../pkg/exec/exec.go#L26-31), which handles external command execution by the `buildImplementation`
+   * The [`s6.Context`](../pkg/s6/s6.go#L23-26), which contains configuration for optionally installing the s6 supervisor to manage the process in the container
+   * Build-time options
+1. Refresh the `build.Context`, which sets initialization and runtime parameters, such as isolation, working directory, the executor and the s3 context.
```

Comment:
```suggestion
1. Refresh the `build.Context`, which sets initialization and runtime parameters, such as isolation, working directory, the executor and the s6 context.
```

Reply:
It is a really small supervisor, only _half_ the size of s6. 😆 

Fixed.